saleor/graphql/account/bulk_mutations/customer_bulk_delete.py

```diff
@@ -53,4 +53,4 @@ def bulk_action(cls, info: ResolveInfo, queryset, /):
         webhooks = get_webhooks_for_event(WebhookEventAsyncType.CUSTOMER_DELETED)
         manager = get_plugin_manager_promise(info.context).get()
         for instance in instances:
-            manager.customer_deleted(instance, webhooks=webhooks)
+            cls.call_event(manager.customer_deleted, instance, webhooks=webhooks)
```

Comment:
Are the newly added `call_event` calls running in transactions?

Reply:
I'm trying to investigate that however, it seems that some mutations work in transactions but I couldn't find place where this transaction starts (some have decorators of with transaction.. statements but that is obvious to find)

Reply:
so we can merge this ticket in this form as it was discussed